src/components/Modal/Modal.vue

```diff
@@ -61,6 +64,10 @@ export default {
     width: {
       type: String,
       default: ''
+    },
+    closeButtonAriaLabel: {
+      type: String,
+      default: 'Close modal'
```

Comment:
I feel like this should be required.  I know it would make this a breaking change but if it's not required it's going to be easy to ignore and not provide and then we'll end up with gaps where we have untranslated text being used.  If we ever do get to where we're internationalizing and we have someone using a screenreader then they'd be hearing labels in a different language than they may understand.  So while a very low probability scenario, if it did happen it would be a pretty bad experience.

Reply:
fair point, updated!